cypress/e2e/buy-shirt.cy.ts

```diff
@@ -1,24 +1,22 @@
-describe("Buy a t-shirt", () => {
-  it("then the t-shirt should be bought", () => {
-    cy.visit("http://automationpractice.com/");
-    cy.get("#block_top_menu > ul > li:nth-child(3) > a").click();
-    cy.get(
-      "#center_column a.button.ajax_add_to_cart_button.btn.btn-default",
-    ).click();
-    cy.get("[style*='display: block;'] .button-container > a").click();
-    cy.get(".cart_navigation span").click();
+import {MenuContentPage, productPage, shoppingCartPage, loginPage, addressStep, shipping, payment} from "../page/index";
+
+const menuContentPage = new MenuContentPage();
 
-    cy.get("#email").type("aperdomobo@gmail.com");
-    cy.get("#passwd").type("WorkshopProtractor");
-    cy.get("#SubmitLogin").click();
-    cy.get(".cart_navigation [type='submit']").click();
-    cy.get("#cgv").click();
-    cy.get(".cart_navigation [type='submit']").click();
-    cy.get(".bankwire").click();
-    cy.get(".cart_navigation [type='submit']").click();
-    cy.get("#center_column > div > p > strong").should(
-      "have.text",
-      "Your order on My Store is complete.",
-    );
+describe("Buy a t-shirt", () => {
+  it("then should be bought a t-shirt", () => {
+    menuContentPage.visitMenuContentPage();
+    menuContentPage.goToTShirtMenu();
+    productPage.addToCart().click();
+    productPage.proceedToCheckout().click();
+    shoppingCartPage.summaryCheckout().click();
+    loginPage.signInEmail();
+    loginPage.signInPassword();
+    loginPage.signInSubmit().click();
+    addressStep.addressCheckout().click();
+    shipping.termsServiceShipping().click();
+    shipping.shippingCheckout().click();
+    payment.payWirePayment().click();
+    payment.confirmOrder().click();
+    payment.completeOrder();
```

Comment:
this method should do one of the following, choose what you want:

1. Rename it as getOrderConfirmation or similar, that should return the confirmation message and perform the assertion here, at the end we should see something like: `payment.getOrderConfirmation().should("have.text", "the confirmation message to verify");`
2. Rename it as validateConfirmationMessage or similar, it should receive the message to validate, at the end we should see something like: `payment.validateConfirmationMessage("the confirmation message to verify")`